src/content/getting-started/prepare-your-provider-account/aws/_index.md

```diff
@@ -2,11 +2,213 @@
+
+Finally, we create an IAM role for Giant Swarm support staff to assume in order to
+access both AWS accounts. This role must have Giant Swarm's account as a trusted
+entity, and we recommend that it enforces multi-factor authentication.
+
+Giant Swarm staff require access to all accounts, so the following steps must be duplicated in both the management cluster and workload cluster accounts.
+
+#### 1. Create the admin policy {#iam-staff-policy}
+
+Go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection and select __Create policy__ to set the admin permissions. Use the [admin JSON policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/admin-role/iam-policy.json) file as the policy content. This time, call the policy
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+#### 2. Create the admin role {#iam-staff-role}
+
+Enter again in the [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection of the AWS console and select __Create role__ When asked to __Select type of trusted entity__ choose __AWS account__.
+
+- In __Account ID__ enter the value `084190472784`.
+- Do not enable __Require external ID__.
+- It is strongly recommended to check the option __Require MFA__ (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
+
+#### 3. Attach policy to role {#iam-staff-role-policy}
+
+Attach the newly created `GiantSwarmAdmin` policy to the role you are creating.
+
+#### 4. Name the role {#iam-staff-role-name}
+
+Name this role:
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+## Step 3: Configure the cluster role identity {#configure-cluster-role-identity}
+
+Remember that `default` AWSClusterRoleIdentity is needed but you can additional add more with specific names. (LETS INCLUDE THIS IN CAPZ TOO)
+
+```need to be updated
+When you want to create a new Cluster API workload cluster in a new AWS account you need to create a CR AWSClusterRoleIdentity which will reference the role in the new AWS Account.
+
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AWSClusterRoleIdentity
+metadata:
+  labels:
+    cluster.x-k8s.io/watch-filter: capi
+  name: ${ROLE_NAME}
+spec:
+  allowedNamespaces:
+    list: null
+    selector: {}
+  roleARN: ${ROLE_ARN}
+  sourceIdentityRef:
+    kind: AWSClusterControllerIdentity
+    name: default
+Where:
+
+${ROLE_NAME} is a short unique name referencing AWS account (ie: development, sandbox or staging2)
+${ROLE_ARN} is full ARN of the role created in previous step with giantswarm-aws-account-prerequisites repository.
+This CR needs to be created only once for each AWS Account. It can be then referenced in the cluster-aws repo in the value aws.awsClusterRoleIdentityName.
+```
+**Note**: In case you are working with a Giant Swarm partner, you might not have access to the platform API. In that case, please provide the role ARNs values, capa controller and staff, to your partner contact.
```

Comment:
```suggestion
**Note**: In case you are working with a Giant Swarm partner, you might not have access to the platform API. In that case, please provide the role ARNs values, CAPA controller and staff to your partner contact.
```